csharp-basic-topics/TaskCompletedTaskFromResultVsReturnInCSharp/TaskCompletedVsTaskFromResultVsReturnInCSharp/ReturnHandler.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace TaskCompletedVsTaskFromResultVsReturnInCSharp;
+
+public class ReturnHandler
+{
+    public async Task<int> UseReturnAsync()
+    {
+        Console.WriteLine($"About to perform some asynchronous work");
```

Comment:
Since you removed the "dynamic" part of the string, there is no need to use `$` anymore.

Reply:
Thank you for picking up that.